examples/abitest/tests/src/tests.rs

```diff
@@ -42,10 +44,32 @@ fn build_wasm() -> std::io::Result<HashMap<String, Vec<u8>>> {
     })
 }
 
+fn load_certificate(certificate: &str) -> anyhow::Result<Certificate> {
```

Comment:
It may as well be in `oak_utils` (or it can be renamed to `oak_common`).

Reply:
I moved it into `oak_runtime::config` (at the moment `oak_utils` is only a `build-dependency` for other things, and it didn't seem worth making it more than that for a three-line helper).